sdk/appcenter-storage/src/test/java/com/microsoft/appcenter/storage/AbstractStorageTest.java

```diff
@@ -146,4 +190,50 @@ private Channel start(Storage storage) {
         storage.onStarted(Mockito.mock(Context.class), channel, "", null, true);
         return channel;
     }
+
+    void verifyTokenExchangeToCosmosDbFlow(
```

Comment:
Thanks @nikitag-ms for the refactoring. 👍 

Looks like `readEndToEndWithNetwork`(https://github.com/Microsoft/AppCenter-SDK-Android/blob/3f0477282ce06938728acab1b85cb8b2d6e565ad/sdk/appcenter-storage/src/test/java/com/microsoft/appcenter/storage/StorageTest.java#L370) could take advantages of this too.